### PR TITLE
Eos-29361: s3 account page design upgrade

### PR DESCRIPTION
**Pull request by sujit-h-singh**
# Problem Statement
- Migration of S3 account page to new design and functionality.

# Design
-  [Figma design](https://www.figma.com/file/PX70qpneKe6HuVezq4hB2a/S3-Account-Screens-Tab-with-S3-Login?node-id=0%3A179)

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [x] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: src/router/index.ts [hunk 1]
```diff
@@ -44,6 +44,15 @@ const routes: Array<RouteConfig> = [
     component: LrAlertDetails,
     props:true
   },
+  {
+    path: "/manage",
+    redirect: "s3Account"
```
Comment (NareshGopal) [inline src/router/index.ts:49]: Why do you redirect this to S3 account page?

File: src/components/manage/s3/LrS3Account.vue [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+                  interface</v-col
+                >
+              </v-row>
+            </v-container>
+          </div>
+        </v-card-text>
+        <v-divider></v-divider>
+        <v-card-actions class="pa-4">
+          <v-btn color="csmprimary" @click="downloadKey()" dark
+            >download & close</v-btn
```
Comment (NareshGopal) [inline src/components/manage/s3/LrS3Account.vue:156]: There was a discussion on this text. Should it be "Download and Close" or "Save and Close"

File: src/router/index.ts (hunk 1 above)
Comment (sujit-h-singh) [reply src/router/index.ts:49]: As if now It is kept as default, after the authorization is implemented we would add logic to route on basis of type of user

File: src/components/manage/s3/LrS3Account.vue (hunk 2 above)
Comment (sujit-h-singh) [reply src/components/manage/s3/LrS3Account.vue:156]: Download and close is final I think.

File: src/router/index.ts (hunk 1 above)
Comment (sujit-h-singh) [reply src/router/index.ts:49]: As discussed the code will go along with user admin page

**Review by NareshGopal — APPROVED** [review]

**Review by ajaysrivas — APPROVED** [review]

**Pull request merged**